crates/phala-trie-storage/src/lib.rs

```diff
@@ -115,3 +167,30 @@ where
             .collect()
     }
 }
+
+#[cfg(feature = "serde")]
+const _: () = {
```

Comment:
It's normal rust statement. Consider `const _: u32 = { /* any number of statements */; 0 };`
use this style rather than `mod xxx {` to save an identifer name and a `use super::*` .

Reply:
thx